grr/server/grr_response_server/gui/ui/components/timestamp/timestamp.scss

```diff
@@ -2,6 +2,7 @@
 
 .timestamp {
   display: inline-block;
+  cursor: pointer;
```

Comment:
IIUC, there is no click action for the whole timestamp. For what purpose do you show `pointer`?

Reply:
Yes, there isn't. I am showing a pointer as per our discussion with melindal@. I showed her a demo of the table and she requested we put a pointer cursor. Maybe there was some confusion? I can re-check or all of us can have a chat and decide what's best.

Reply:
Please remove this for now, then your PR is ready to merge!